src/client/reducers/index.js

```diff
@@ -1,8 +1,10 @@
 import {combineReducers} from 'redux';
 import postUserDataReducer from './postUserDataReducer';
 import getUserDataReducer from './getUserDataReducer';
+import audioreducer from './audioReducer';
```

Comment:
Please rename this reducer, by using camel case

Reply:
ok